packages/ui-kit/src/components/rss-feed-table.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@emotion/css';
+import moment from 'moment';
+import Link from './link';
+import ContentNotifications from './content-notifications';
+import { colors, tokens, dimensions } from '../design-system';
+
+const Table = styled.table`
+  width: 100%;
+  border-radius: ${tokens.borderRadiusForRssFeedTable};
+  box-shadow: ${tokens.shadowForRssFeedTable};
+
+  thead {
+    th {
+      background-color: ${colors.light.surfaceSecondary2};
+      border-radius: ${tokens.borderRadiusForRssFeedTable}
+        ${tokens.borderRadiusForRssFeedTable} 0 0;
+      padding: ${dimensions.spacings.s};
+    }
+  }
+
+  tbody {
+    tr {
+      td {
+        padding: ${dimensions.spacings.m};
+
+        :first-of-type {
+          color: ${colors.light.linkHover};
+        }
+      }
+
+      :not(:first-of-type) {
+        border-top: 1px solid ${colors.light.borderPrimary};
+      }
+    }
+  }
+`;
+
+const DateWrapper = styled.span`
+  white-space: nowrap;
+`;
+
+const RssFeedTable = (props) => {
+  if (!props.data) {
+    const message = 'Must pass data to RssFeedTable component.';
+    if (process.env.NODE_ENV !== 'production') {
+      return <ContentNotifications.Error>{message}</ContentNotifications.Error>;
+    }
+
+    throw new Error(message);
+  }
+
+  return (
+    <div>
+      <Table>
+        <thead>
+          <tr>
+            <th colSpan="3">Release Notes</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.data.map((item, index) => (
+            <tr key={`${item.title}${index}`}>
+              <td
+                css={css`
+                  width: ${dimensions.widths.rssFeedDateWidth};
```

Comment:
Please do not use fixed pixel widths but rather let the browser optimize and prevent word wrapping the first two columns by giving them  `white-space: nowrap;`.  That's generally better layout and also more robust in small screen widths.  

The preview layout uses too much white space in the first two columns while still breaking the third into multiple lines.

Reply:
👍